Functions.cmake

```
@@ -70,6 +70,8 @@ function(install_js_thrift)
     DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/gen-js/
     DESTINATION ${INSTALL_GEN_DIR}
     FILES_MATCHING PATTERN "*.js")
+  install(
+    CODE "execute_process(COMMAND ${CMAKE_SOURCE_DIR}/scripts/remover.sh WORKING_DIRECTORY ${INSTALL_GEN_DIR})")
```

```suggestion
    CODE "execute_process(COMMAND /bin/bash ${CMAKE_SOURCE_DIR}/scripts/remover.sh WORKING_DIRECTORY ${INSTALL_GEN_DIR})")
```
When omitting `/bin/bash`, the *remover.sh* was not executed for me, even after correcting the shebang (see below).

---

Uh I really missed typing again... I fixed the shebang.
The install work just fine on my PC without the `/bin/bash` , I don't know what makes the difference.

---

Maybe it can be because you use a different shell from bash?
If the problem is caused by that I'll add the `/bin/bash` to the script.

---

The default shell for my user is `/bin/bash`, so it should not have been an issue.
I will test it on another computer next week.

---

Well, it worked on another machine now (with Ubuntu Bionic). I will merge this now and if any issue is raised, I will add `/bin/bash` later.